### PR TITLE
Added xmlrpc dependency for ruby >= 2.4.0 support

### PR DESCRIPTION
**Pull request by mysticaltech**
Hey man, that's the only change needed and it works well on 2.4.0.

### CODE REVIEW

**Comment by zachfeldman** [comment]
Very cool. Would you mind also doing a version bump and adding yourself to the contributors list? Then I'll redeploy to Rubygems so we can support 2.4.0!

Thanks again @rimkashox .

**Comment by rimkashox** [comment]
My pleasure @zachfeldman!

File: README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ To make further requests, check out the documentation - this gem should follow t
 ## Contributing to rubypress
 
 Pull requests welcome.
- 
+
```
Comment (zachfeldman) [inline README.md]: I'm seeing a lot of extra details in this commit, mind removing these?

File: README.md [hunk 2]
```diff
@@ -190,12 +190,13 @@ variable names include `HTTP_` and can be set to the same or different values as
 * Casey Hadden [@caseyhadden](https://github.com/caseyhadden) (Added support for cookie-based authentication schemes)
 * Noah Botimer [@botimer](https://github.com/botimer) (Allowed custom prefixes on method names and tests to run against https servers on any port)
 * Carlos Pérez Cerrato [@lastko](https://github.com/lastko) (Caught Errno::EPIPE: Broken pipe errors)
-* Eric Gascoine [@ericgascoine](https://github.com/ericgascoine) Fixed getPostStatusList
-* Matt Colyer [@mcoyler](https://github.com/mcolyer) Added configurable timeouts.
+* Eric Gascoine [@ericgascoine](https://github.com/ericgascoine) (Fixed getPostStatusList)
+* Matt Colyer [@mcoyler](https://github.com/mcolyer) (Added configurable timeouts)
+* Karim Naufal [@mcoyler](https://github.com/rimkashox) (Added support for Ruby >= 2.4.0)
```
Comment (zachfeldman) [inline README.md]: Almost there, you stole @mcoyler 's username!

File: rubypress.gemspec [hunk 3]
```diff
@@ -2,7 +2,7 @@ $LOAD_PATH.push File.expand_path('../lib', __FILE__)
 
 Gem::Specification.new do |s|
   s.name = 'rubypress'
-  s.version = '1.2.1'
+  s.version = '1.2.2'
```
Comment (zachfeldman) [inline rubypress.gemspec]: Good

**Review by zachfeldman — CHANGES_REQUESTED** [review]
Sorry, just a few minor edits needed before merge.

File: README.md (hunk 1 above)
Comment (rimkashox) [inline README.md]: Hey man, this must be my atom text editor that's doing some automatic fixing of blank lines (it's the editor from Github, just an amazing piece of code). I didn't manually do anything, and if you look at it carefully, you will see that nothing really happens.

File: rubypress.gemspec (hunk 3 above)
Comment (rimkashox) [inline rubypress.gemspec]: 👍🏻

**Comment by rimkashox** [comment]
Zach, all should be good now, no? Would really be cool to have the Rubypress fully compatible with 2.4.0... I really don't care about having my name in the readme, tell me if you'd like me to revert this change!

File: README.md (hunk 1 above)
Comment (zachfeldman) [inline README.md]: Sure, I use Atom too. I'll ignore this for now but I'd advise you to watch out for projects that are more strict on commits!

**Pull request merged**